lib/prismic/fragments/link.rb

```diff
@@ -3,16 +3,20 @@ module Prismic
   module Fragments
     class Link < Fragment
 
-      def start_html(link_resolver = nil)
-        %(<a href="#{url(link_resolver)}">)
+      def start_html(link_resolver = nil, target = nil)
+        unless target.nil?
+          %(<a href="#{url(link_resolver)}" target="#{target}" rel="noopener">)
+        else 
+          %(<a href="#{url(link_resolver)}">)
+        end
       end
 
       def end_html
         %(</a>)
       end
 
       def as_html(link_resolver=nil)
-        %(#{start_html(link_resolver)}#{url(link_resolver)}#{end_html})
+        %(#{start_html(link_resolver, target)}#{url(link_resolver)}#{end_html})
```

Comment:
Where does `target` come from?

Reply:
It comes from any of the sub classes. I can change it to `@target` if that makes it more clear. It works either way.

Reply:
Ok It think it would be more clear.